content/docs/command-reference/get.md

```diff
@@ -36,12 +36,12 @@ the data source. Both HTTP and SSH protocols are supported for online repos
 to an "offline" repo (if it's a DVC repo without a default remote, instead of
 downloading, DVC will try to copy the target data from its <abbr>cache</abbr>).
 
-The `path` argument is used to specify the location of the target to be
-downloaded within the source repository at `url`, including those nested inside
-directories [tracked as a whole](/doc/command-reference/add#example-directory).
-`path` can specify any file or directory in the source repo, either tracked by
-DVC, or by Git. Note that DVC-tracked targets should be found in a `dvc.yaml` or
-`.dvc` file of the project.
+The `path` argument is used to specify the location of the target to download
+within the source repository at `url`, including those nested inside
+[tracked directories](/doc/command-reference/add#tracking-directories). `path`
```

Comment:
I still think this link can be confusing and doesn't cover `dvc.yaml`. 

in this specific case should "including part" go in the next sentence?

Reply:
OK. Removed the link and simplified the note (in all the docs) in 41f4db7f.